lib/chai/utils/getOwnEnumerablePropertySymbols.js

```diff
@@ -0,0 +1,27 @@
+/*!
+ * Chai - getOwnEnumerablePropertySymbols utility
+ * Copyright(c) 2011-2016 Jake Luer <jake@alogicalparadox.com>
+ * MIT Licensed
+ */
+
+/**
+ * ### .getOwnEnumerablePropertySymbols(object)
+ *
+ * This allows the retrieval of directly-owned enumerable property symbols of an
+ * object. This function is necessary because Object.getOwnPropertySymbols
+ * returns both enumerable and non-enumerable property symbols.
```

Comment:
This description seems wrong since this function returns only enumerable symbol properties and not all of them (enumerable and non-enumerable)